src/libraries/Microsoft.PowerFx.Core/Functions/TexlFunction.cs

```diff
@@ -1738,5 +1740,39 @@ internal ArgPreprocessor GetGenericArgPreprocessor(int index)
 
             return ArgPreprocessor.None;
         }
+
+        /// <summary>
+        /// Visit all function nodes to compose dependency info.
+        /// </summary>
+        /// <param name="node">IR CallNode.</param>
+        /// <param name="visitor">Dependency visitor.</param>
+        /// <param name="context">Dependency context.</param>
+        /// <returns>Static boolean value.</returns>
+        public virtual bool ComposeDependencyInfo(IRCallNode node, DependencyVisitor visitor, DependencyContext context)
+        {
+            for (int i = 0; i < node.Args.Count; i++)
+            {
+                if (node.Scope != null && i < node.Function.ScopeArgs)
+                {
+                    if (node.Args[i] is IRCallNode callNode)
+                    {
+                        callNode.Accept(visitor, context);
+                    }
+                    else
+                    {
+                        continue;

```

Comment:
>continue; [](http://example.com/codeflow?start=24&length=9)

why do we not need to visit this node?
I.e. something like ShowColumns(), can have TextLiteralNode which are column name being used, Are we skipping those?

Reply:
New logic will visit.